### PR TITLE
Change example Job Admission Policy to use matchLabels

### PR DESCRIPTION
**Pull request by dgrove-oss**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

Making the example policy be "opt-in" instead of "opt-out" seems more likely to be useful to users trying to adapt the example to their own use cases. 

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

**Review by mimowo — COMMENTED** [review]
/lgtm
Another possibility is to keep it "opt-out" using labels, say "kueue-unmanaged", but I think both approaches are equally valid. 
/assign @alculquicondor 
For final tagging.

**Comment by dgrove-oss** [comment]
Did something change in the CLA tooling?  I've been on the IBM corporate CLA with CNCF to contribute to Kubernetes for multiple years; not sure why it is failing now.

File: site/static/examples/sample-validating-policy.yaml [hunk 1]
```diff
@@ -15,5 +15,5 @@ spec:
       operations:  ["CREATE", "UPDATE"]
       resources:   ["jobsets"]
   validations:
-    - expression: "'kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
+    - expression: "has(object.metadata.labels) && 'kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''"
```
Comment (dgrove-oss) [inline site/static/examples/sample-validating-policy.yaml:18]: Avoid getting a lower-level error message when object does not have labels.
Without the extra check you get:
```
The jobs "sample-job" is invalid: : ValidatingAdmissionPolicy 'mlbatch-require-queue-name' with binding 'mlbatch-require-queue-name' denied request: expression ''kueue.x-k8s.io/queue-name' in object.metadata.labels && object.metadata.labels['kueue.x-k8s.io/queue-name'] != ''' resulted in error: no such key: labels
```

File: site/content/en/docs/tasks/manage/setup_job_admission_policy.md [hunk 2]
```diff
@@ -45,8 +45,14 @@ To create the binding, download the above file and run the following command:
 kubectl create -f sample-validating-policy-binding.yaml
 ```
 
-Now, when you try to create a `Job` or a `JobSet` without the `kueue.x-k8s.io/queue-name` label or value in any namespace other than `admin-namespace`,
-the error message will be similar to the following:
+Label each namespace where you want this policy to be enforced by doing:
```
Comment (alculquicondor) [inline site/content/en/docs/tasks/manage/setup_job_admission_policy.md:48]:
```suggestion
Run the following command to label each namespace where you want this policy to be enforced:
```

**Pull request merged**